examples/platforms/simulation/radio.c

```diff
@@ -829,7 +831,10 @@ void platformRadioProcess(otInstance *aInstance, const fd_set *aReadFdSet, const
 
         if (rval > 0)
         {
-            if (sockaddr.sin_port != htons(sPort))
+            uint16_t srcPort   = ntohs(sockaddr.sin_port);
+            uint16_t srcNodeId = srcPort - sPortOffset - 9000;
```

Comment:
Port start is not always 9000. It should be calculated from `PORT_OFFSET` ?

Reply:
I have already considered the port offset with `sPortOffset`. The offset is relative to 9000, and 9000 is hard encoded in this file elsewhere several times.

Reply:
SGTM.